src/timestamp.rs

```diff
@@ -0,0 +1,74 @@
+// LNP/BP Core Library implementing LNPBP specifications & standards
+// Written in 2020 by
+//     Dr. Maxim Orlovsky <orlovsky@pandoracore.com>
+//
+// To the extent possible under law, the author(s) have dedicated all
+// copyright and related and neighboring rights to this software to
+// the public domain worldwide. This software is distributed without
+// any warranty.
+//
+// You should have received a copy of the MIT License
+// along with this software.
+// If not, see <https://opensource.org/licenses/MIT>.
+use chrono::{Utc, Local, DateTime, TimeZone};
+use std::ops::{Add, Sub};
+
+#[derive(Clone, Debug, PartialEq, Hash, Eq, PartialOrd, Ord)]
+pub struct Timestamp(pub u32);
```

Comment:
> Actually, can you pls rebase on the current master, since I just merged #8 - and put this module not in `lightning_encoding` but into `lnp2p/src/legacy/types.rs`?

Hi @dr-orlovsky yes, I'm back at home and recovering from a weird bitcoiner virus I got 😄 , I am starting to work on this, thanks for the feedback